guestbook/php-redis/Makefile

```diff
@@ -0,0 +1,91 @@
+# Copyright 2017 The Kubernetes Authors. All rights reserved.
```

Comment:
you add a Makefile but there is no reference to it anywhere. Not in the README. So how is the user supposed to know what to do with this Makefile ?